ts/components/session/SessionRegistrationView.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { AccentText } from './AccentText';
+
+import { LocalizerType } from '../../types/Util';
+import { RegistrationTabs } from './RegistrationTabs';
+
+declare global {
+  interface Window {
+    displayNameRegex: any;
```

Comment:
Unused? It is also already declared in `global.d.ts`.

Reply:
correct